srv/travel-service.js

```diff
@@ -28,7 +28,7 @@ init() {
    * upon SAVE, as multiple users could create new Travels concurrently.
    */
   this.before ('CREATE', 'Travel', async req => {
-    const { maxID } = await SELECT.one `max(TravelID) as maxID` .from (Travel)
+    const { maxID } = await SELECT.one `max(TravelID) as maxID` .from (Travel) .forUpdate()
```

Comment:
I think `forUpdate` is not allowed on aggregations as there are not actual rows returned that could be locked.

Reply:
Indeed, it's not allowed for aggregate functions. Let's close this PR.